scopes/component/ui/version-dropdown/version-dropdown.tsx

```diff
@@ -73,17 +108,76 @@ export function VersionDropdown({
             latestVersion={latestVersion}
             localVersion={localVersion}
             currentLane={currentLane}
+            overrideVersionHref={overrideVersionHref}
+            showVersionDetails={showVersionDetails}
           />
         )}
       </Dropdown>
     </div>
   );
 }
 
-function VersionPlaceholder({ currentVersion, className }: { currentVersion?: string; className?: string }) {
+type VersionPlaceholderProps = {
+  showDetails?: boolean;
+  currentVersion: string;
+  tags: DropdownComponentVersion[];
+  snaps?: DropdownComponentVersion[];
+  disabled?: boolean;
+} & HTMLAttributes<HTMLDivElement>;
+
+function VersionPlaceholder({
```

Comment:
extract

Reply:
split into SimpleVersion and FullVersion instead of <VersionPlaceholder showDetails/>